docs/guides/infineon_trustm_provisioning.md

```diff
@@ -39,12 +41,12 @@ can be used to perform provisioning by following the steps mentioned below.
 
 ```
 $ cd scripts/matter_provisioning/
-$ ./matter_dac_provisioning.sh
+$ ./matter_test_provisioning.sh
 ```
 
 _Note:_
 
-_By running this example matter_dac_provisioning.sh, the steps shown below are
+_By running this example matter_test_provisioning.sh, the steps shown below are
```

Comment:
```suggestion
_By running this example `matter_test_provisioning.sh`, the steps shown below are
```

Reply:
Sure. I will modify it accordingly. Thanks for your comments.